clients/instance/ibm-pi-network-address-group.go

```diff
@@ -24,6 +24,10 @@ func NewIBMPINetworkAddressGroupClient(ctx context.Context, sess *ibmpisession.I
 
 // Create a new Network Address Group
 func (f *IBMPINetworkAddressGroupClient) Create(body *models.NetworkAddressGroupCreate) (*models.NetworkAddressGroup, error) {
+	// Add check for on-prem location
+	if f.session.IsOnPrem() {
+		return nil, fmt.Errorf("not supported in on-prem location")
```

Comment:
```suggestion
		return nil, fmt.Errorf("operation not supported in satellite location, check documentation")
```
this is a slightly differently worded message than the ones currently in use

I don't have a preference either way, but I would like all of them to be consistent. Maybe create a constant for it?